helm/examples/dev/Chart.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v2
+type: application
+name: aleph-dev
+version: 0.1.0
+dependencies: 
+  - name: aleph
+    repository: file://../../charts/aleph
+    version: 3.15.4
```

Comment:
I'm not sure if I did something wrong, but I get aleph `3.15.1` :/

Reply:
Hm, that’s weird… I just reset my cluster, removed local images, and set up everything again from scratch. But looking at the about page and at the version specified inside of the container I see `3.15.4`. We could look into it together tomorrow if you’d like :)

Reply:
Only other reason for this I could imagine is that this branch used to be based off of 3.15.1, and I later rebased it and force pushed to be up to date with develop. But that was a few days before you looked at it, I think…